decorators/capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.CapitalizeDecorator
```

Comment:
- [ ] CapitalizeDecorator method is not defined. May you want to say `capitalize`? Capitalize method capitalize the first letter of the given name.